src/extension/importer/core/src/main/java/org/geoserver/importer/Importer.java

```diff
@@ -109,6 +114,14 @@
 public class Importer implements DisposableBean, ApplicationListener {
 
     static Logger LOGGER = Logging.getLogger(Importer.class);
+    
+    public static final String PROPERTYFILENAME = "geoserver-importer.properties";
```

Comment:
Other plugins using simple config files are not prefixing with "geoserver-", like "controlflow.properties".
Probably better to just call it "importer.properties"

Reply:
Ok. Also might be good to have the importer config directory as you suggested on the other PR.